boilerplate/app.json.ejs

```diff
@@ -4,7 +4,7 @@
     "description": "No description",
     "slug": "<%= props.name %>",
     "privacy": "public",
-    "sdkVersion": "25.0.0",
+    "sdkVersion": "30.0.0",
```

Comment:
I was trying to keep the repo `package.json` version in sync with the expo version so that should be bumped as well.